MichaelEWalker87/Travel-Tracker
# Mobile view

# Pull Request Travel-Tracker



#### What’s this PR do?

* This sets up a base mobile view for the user

#### Where should the reviewer start?

* user should start with the SASS media queries 

#### How should this be manually tested?

* Open up the browser and place the application into mobile view the application should adjust to the changes 

#### Any background context you want to provide?

* I origanly thought I would have to remove the icons for space however this was not the case 

#### What are the relevant tickets?

*  need to remove the mobile icons from the HTML and JS

#### Screenshots (if appropriate):

![mobile login](https://i.imgur.com/QhPkA5u.png)

#### Questions:

* not at this time
